bears/python/BanditBear.py

```diff
@@ -47,7 +55,22 @@ def create_arguments(filename, file, config_file,
                       'LOW': 50}
 
     def process_output(self, output, filename, file):
-        output = json.loads(output)
+        def warn_issue(message):
+            self.warn('While running {0}, some issues were found:'
+                      .format(self.__class__.__name__))
+            self.warn(message)
+
+        # Taking output from stderr in case bandit shows errors
+        # such as selected test ID and skipped test ID are same.
+        err_pattern = re.compile(r'ERROR.*')
+        match = err_pattern.search(output[1])
+        if match:
+            warn_issue(match.group())
+
+        if not file or not output[0]:
+            return
+
+        output = json.loads(output[0])
```

Comment:
The try, catch block was removed from here ?
(Not sure though if bandit will ever give incorrect json, so just asking :sweat_smile: )

Reply:
try, catch block was never here, though I tried generating one incorrect json using 6/7 test files and in all the case, it either gave ERROR (which was not in json) or json output. So, IMO it is not required.

Reply:
Ah
Cool :+1: